### DISCUSSION TITLE
Kesulitan mencari optimal number of clusters k-means

### DISCUSSION THREAD

**Discussion by reyhanpurnawan** [teamalgoritma/community#489] [Q&A] [answered]
Selamat siang, saya kesulitan mencari k optimum saat akan melakukan clustering k-means. Dari tiga metode elbow, silhouette, gap, mengeluarkan k yang berbeda-beda. Apakah bisa dibantu?
<img width="1440" alt="Screenshot 2022-12-16 at 12 14 36" src="https://user-images.githubusercontent.com/88662517/208027949-b8c45f13-39cf-4184-aaa7-0187387564e6.png">
<img width="1440" alt="Screenshot 2022-12-16 at 12 14 29" src="https://user-images.githubusercontent.com/88662517/208027970-a26ddb57-1eac-4e0a-a565-8602521a58b3.png">
<img width="1440" alt="Screenshot 2022-12-16 at 12 14 33" src="https://user-images.githubusercontent.com/88662517/208027974-70b0cfc1-ed5b-45d5-9d15-0cf69d53bbf2.png">
[Angkasa Pura Lombok.xlsx](https://github.com/teamalgoritma/community/files/10242867/Angkasa.Pura.Lombok.xlsx)
berikut datasetnya
Terima kasih

**Comment by yosiaazarya** [comment, marked as answer]
Halo pak, seperti yang pernah kita pelajari di kelas UL bahwa mungkin saja beberapa metode dapat megnhasilkan nilai optimal cluster yang berbeda. Namun kita dapat membuat beberapa hal dengan K = 2 , 3 , atau 5. Serta dapat mendapatkan bantuan dari penggunaan biplot untuk dapat mengetahui persebaran dari cluster-cluster yang terbentuk.

**Reply by reyhanpurnawan** [reply]
baik terima kasih mas
